tests/std/tests/GH_002307_usual_scope_guard/test.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#define _CORE_HEADERS_ONLY
+
+#include <cassert>
+#include <deque>
+#include <stdexcept>
+
+using namespace std;
+
+struct bomb {

```

Comment:
It fits well, but I am not sure if i really like bombs in code bases. Usually we call this Evil or something.

Reply:
💥💥💥

Reply:
Definitely don't like bombs, anywhere.